Validation/GlobalHits/src/GlobalHitsAnalyzer.cc

```diff
@@ -14,6 +14,10 @@
 #include "Geometry/HcalCommonData/interface/HcalHitRelabeller.h"
 #include "Geometry/HcalTowerAlgo/interface/HcalGeometry.h"
 #include "Geometry/Records/interface/HcalRecNumberingRecord.h"
+#include "FWCore/Framework/interface/GetterOfProducts.h"
+#include "FWCore/Framework/interface/ProcessMatch.h"
+
+edm::GetterOfProducts<edm::HepMCProduct> getterOfProducts_;
```

Comment:
Shouldn't this be a private member of the class `GlobalHitsAnalyzer` like, for example, `tGeomToken_` ?
https://github.com/cms-sw/cmssw/blob/d32bfd821fd0a23b7eebb354fc51b2f7917962b9/Validation/GlobalHits/interface/GlobalHitsAnalyzer.h#L147